src/test/skript/tests/regressions/pull-5748-fix bugs in the random number expression.sk

```diff
@@ -0,0 +1,8 @@
+
+# Test not related to a made issue. Details at pull request.
```

Comment:
pull request # here? otherwise people are gonna have to search through the blame

Reply:
The pull request's id is in the file name

Reply:
maybe still worth including imo (redundancy cant hurt!)